2023-04-30T12:21:55.8382832+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2023-04-30T12:21:55.8584395+05:30  [INF] Hosting environment: "Development" (c3307c92)
2023-04-30T12:21:55.8607111+05:30  [INF] Content root path: "C:\MyProject\Vehicle-mishap\ODPortalWebAPI" (b5d60022)
2023-04-30T12:21:55.9591233+05:30 80000036-0005-f800-b63f-84710c7967bb [WRN] Failed to determine the https port for redirect. (ca76cc21)
2023-04-30T12:28:36.4302700+05:30 80000004-0007-ff00-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
ODPortalWebDL.DTO.ExceptionModal.CustomException: Invalid Login details entered.
   at ODPortalWebDL.DataAccess.UserManagerDataAccess.CheckUserPassWord(Credentials credentials) in C:\MyProject\Vehicle-mishap\ODPortalWebDL\DataAccess\UserManagerDataAccess.cs:line 63
   at ODPortalWebDL.Manager.UserManager.Auth(Credentials credentials) in C:\MyProject\Vehicle-mishap\ODPortalWebDL\Manager\UserManager.cs:line 26
   at ODPortalWebAPI.Controllers.UserController.Authenticate(Credentials credentials) in C:\MyProject\Vehicle-mishap\ODPortalWebAPI\Controllers\UserController.cs:line 30
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at ODPortalWebAPI.Startup.<>c.<<Configure>b__5_1>d.MoveNext() in C:\MyProject\Vehicle-mishap\ODPortalWebAPI\Startup.cs:line 59
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2023-04-30T12:28:45.0093217+05:30 80000006-0007-ff00-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
ODPortalWebDL.DTO.ExceptionModal.CustomException: Invalid Login details entered.
   at ODPortalWebDL.DataAccess.UserManagerDataAccess.CheckUserPassWord(Credentials credentials) in C:\MyProject\Vehicle-mishap\ODPortalWebDL\DataAccess\UserManagerDataAccess.cs:line 63
   at ODPortalWebDL.Manager.UserManager.Auth(Credentials credentials) in C:\MyProject\Vehicle-mishap\ODPortalWebDL\Manager\UserManager.cs:line 26
   at ODPortalWebAPI.Controllers.UserController.Authenticate(Credentials credentials) in C:\MyProject\Vehicle-mishap\ODPortalWebAPI\Controllers\UserController.cs:line 30
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at ODPortalWebAPI.Startup.<>c.<<Configure>b__5_1>d.MoveNext() in C:\MyProject\Vehicle-mishap\ODPortalWebAPI\Startup.cs:line 59
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2023-04-30T12:29:52.9475732+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2023-04-30T12:29:52.9650716+05:30  [INF] Hosting environment: "Development" (c3307c92)
2023-04-30T12:29:52.9670574+05:30  [INF] Content root path: "C:\MyProject\Vehicle-mishap\ODPortalWebAPI" (b5d60022)
2023-04-30T12:29:53.0539471+05:30 80000006-0006-fc00-b63f-84710c7967bb [WRN] Failed to determine the https port for redirect. (ca76cc21)
2023-04-30T12:30:47.8164320+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2023-04-30T12:30:47.8647934+05:30  [INF] Hosting environment: "Development" (c3307c92)
2023-04-30T12:30:47.8719080+05:30  [INF] Content root path: "C:\MyProject\Vehicle-mishap\ODPortalWebAPI" (b5d60022)
2023-04-30T12:30:48.0075885+05:30 8000000d-0007-ff00-b63f-84710c7967bb [WRN] Failed to determine the https port for redirect. (ca76cc21)
2023-04-30T12:30:59.9101919+05:30 80000039-0005-f800-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
ODPortalWebDL.DTO.ExceptionModal.CustomException: Invalid Login details entered.
   at ODPortalWebDL.DataAccess.UserManagerDataAccess.CheckUserPassWord(Credentials credentials) in C:\MyProject\Vehicle-mishap\ODPortalWebDL\DataAccess\UserManagerDataAccess.cs:line 63
   at ODPortalWebDL.Manager.UserManager.Auth(Credentials credentials) in C:\MyProject\Vehicle-mishap\ODPortalWebDL\Manager\UserManager.cs:line 26
   at ODPortalWebAPI.Controllers.UserController.Authenticate(Credentials credentials) in C:\MyProject\Vehicle-mishap\ODPortalWebAPI\Controllers\UserController.cs:line 30
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at ODPortalWebAPI.Startup.<>c.<<Configure>b__5_1>d.MoveNext() in C:\MyProject\Vehicle-mishap\ODPortalWebAPI\Startup.cs:line 59
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2023-04-30T12:32:40.6045849+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2023-04-30T12:32:40.7472468+05:30  [INF] Hosting environment: "Development" (c3307c92)
2023-04-30T12:32:40.7601657+05:30  [INF] Content root path: "C:\MyProject\Vehicle-mishap\ODPortalWebAPI" (b5d60022)
2023-04-30T12:32:41.0124455+05:30 8000004a-0004-fd00-b63f-84710c7967bb [WRN] Failed to determine the https port for redirect. (ca76cc21)
2023-04-30T12:34:40.4245326+05:30 8000003e-0005-f800-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
ODPortalWebDL.DTO.ExceptionModal.CustomException: Invalid Login details entered.
   at ODPortalWebDL.DataAccess.UserManagerDataAccess.CheckUserPassWord(Credentials credentials) in C:\MyProject\Vehicle-mishap\ODPortalWebDL\DataAccess\UserManagerDataAccess.cs:line 63
   at ODPortalWebDL.Manager.UserManager.Auth(Credentials credentials) in C:\MyProject\Vehicle-mishap\ODPortalWebDL\Manager\UserManager.cs:line 26
   at ODPortalWebAPI.Controllers.UserController.Authenticate(Credentials credentials) in C:\MyProject\Vehicle-mishap\ODPortalWebAPI\Controllers\UserController.cs:line 30
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at ODPortalWebAPI.Startup.<>c.<<Configure>b__5_1>d.MoveNext() in C:\MyProject\Vehicle-mishap\ODPortalWebAPI\Startup.cs:line 59
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.<Invoke>g__AwaitMatcher|8_0(EndpointRoutingMiddleware middleware, HttpContext httpContext, Task`1 matcherTask)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
